packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```diff
@@ -103,7 +103,16 @@ cp -pa .%{gem_dir}/* \
 %{foreman_restart}
 exit 0
 
+%post
+semanage port -a -t http_port_t -p tcp 8006 &> /dev/null || :
+
+%postun
+semanage port -d -t http_port_t -p tcp 8006 &> /dev/null || :
```

Comment:
I thought postun was also called on upgrades. This means it'll flip flop between on/off/on. https://www.golinuxhub.com/2018/05/how-to-execute-script-at-pre-post-preun-postun-spec-file-rpm.html has some examples on how to check for this.

Reply:
I guess, you are the expert on rpm packaging questions. `postun` sounded right to me, so i had no reason to question that. Feel free to add any needed change to this PR.

Reply:
I think technically an upgrade is an uninstall + install so it's called in both removals + upgrades. I also wouldn't consider myself an expert ;)

Reply:
maybe prepending `[ $1 == 0 ] && ` is enough.

Reply:
When i read the article correctly, it is even install + uninstall. So my version would have left the port closed on update.
Thank you both for spotting and repairing this.